src/backy/tests/test_daemon.py

```diff
@@ -76,6 +77,10 @@ def test_reload(daemon, tmpdir):
 ---
 global:
     base-dir: {new_base_dir}
```

Comment:
I don't see specific tests to verify our authentication code. I'd like to have at least full branch coverage for the authentication function covered on this level.